README.md

```diff
@@ -78,9 +78,9 @@ to any issue or pull request you'd like to see worked on. Thanks.
 
 1. Download a pre-compiled, released version from the [Consul Template releases page][releases].
 
-1. Extract the binary using `unzip` or `tar`.
+2. Extract the binary using `unzip` or `tar`.
 
-1. Move the binary into `$PATH`.
+3. Move the binary into `$PATH`.
```

Comment:
It's okay to use `1.` for every item in a list when using Markdown. The items will be correctly numbered when the document is rendered. See <https://github.com/hashicorp/consul-template?tab=readme-ov-file#installation> .